lib/gruf/hooks/base.rb

```diff
@@ -45,6 +45,22 @@ def initialize(service, options = {})
       def setup
         # noop
       end
+
+      ##
+      # @return [String] Returns the service name as a translated name separated by periods
+      #
+      def service_key
+        service.class.name.underscore.tr('/', '.')
+      end
+
+      ##
+      # Parse the method signature into a service.method name format
+      #
+      # @return [String] The parsed service method name
+      #
+      def method_key(call_signature)
```

Comment:
It seems redundant to log the service_key an extra time here.

Here is an example log from a health check service I have running.

```json
{
  "params":{},
  "message":"",
  "service":"health.health_check_service",
  "action":"health.health_check_service.check",
  "method":"health.health_check_service.check",
  "grpc_status":"GRPC::Ok",
  "duration":3.5,
  "status":0,
  "thread_id":46958605007800,
  "time":"2017-08-01 14:27:48 +0000",
  "host":"ebfc65cb3169",
  "format":"json",
  "@timestamp":"2017-08-01T14:27:48.405+00:00",
  "@version":"1",
  "severity":"INFO"
}
```

I think action and method should both just be the gRPC method name without being prefixed with the service_key.

Thoughts?